### ISSUE TITLE
Update alt text for gender-equality image on the About Us page to adhere to WCAG

### ISSUE THREAD

**Issue by kathrynsilvaconway** [hackforla/website#3383]
### Overview
We need to ensure that alt text for images adhere to Web Content Accessibility Guidelines (WCAG). In this issue, we will change the alt text for the Radical Inclusivity image on the [About Us page](https://www.hackforla.org/about/).

### Details
Designers and PMs identified that some alt texts in the Events page should be changed because they are either inaccurate or too long. Ideally, alt text should be descriptive but short and succinct. According to the WCAG page, an accompanying alt text should be short. Additionally, the Functional Accessibility Evaluator (FAE) recommends for alt text to be no more than 100 characters.

### Action Items
- [x] In the file `__includes/about-page/about-card-sustainability.html`, change the following alt text

From
```
Gender Equality goal logo
```

To
```
5 Gender Equality
```

- [ ] **For merge team**: Once the pull request associated with this issue is approved and merged, please update and edit epic #3361 by
  - [ ] Checking off the dependency for this issue
  - [ ] If all dependencies are checked off, please move issue #3361 to the New Issue Approval column and remove the Dependency label

### Resources/Instructions
- This issue is part of epic #3361 
- [About Us page](https://www.hackforla.org/about/)
- [Alt text decision Tree ](https://www.w3.org/WAI/tutorials/images/decision-tree/)
- [FAE rule 4: alt text should be less than 100 characters](https://fae.disability.illinois.edu/rulesets/IMAGE_4_EN/)

#### File and code links you will need to work on this issue
- Directory to find the page once you have it in your IDE: `_includes/about-page/about-card-platform.html`
- Click on the link below to see the source file code:
https://github.com/hackforla/website/blob/a83dd650b664618ff2352b2fb93e8ae4acfb402b/_includes/about-page/about-card-platform.html#L77

#### Information about WCAG
- [WCAG](https://www.w3.org/WAI/standards-guidelines/wcag/)
- [Web Accessibility Tutorials: Decorative Images](https://www.w3.org/WAI/tutorials/images/decorative/)
- [WCAG section on alt text](https://www.w3.org/WAI/WCAG21/Techniques/html/H37.html)

**Comment by Adaakal** [comment]
**Progress:** 
**What is the current status of your project?** 
Alt text has been changed. 
**What have you completed and what is left to do?**
Git-related tasks
